packages/esm-form-entry-app/src/app/fe-wrapper/fe-wrapper.component.ts

```diff
@@ -231,8 +237,8 @@ export class FeWrapperComponent implements OnInit, OnDestroy {
             showToast({
               critical: true,
               kind: 'success',
-              description: `The form has been submitted successfully.`,
-              title: this.form.schema.name,
+              description: this.translateService.instant('formSubmittedSuccessfully'),
+              title: this.form.schema.display,
```

Comment:
There may be something we need to do here, but I don't think this is it. Specifically, `this.form.schema` refers to the schema itself, which does not have a display property. At least `name` will be defined.